.circleci/conditional_config.yml

```diff
@@ -251,8 +244,12 @@ jobs:
           root: .
           paths:
             - src/api/coverage_results
-      - <<: *store_rspec_artifacts
-      - <<: *store_rspec_test_results
+      - store_test_results:
+          path: /home/frontend/rspec
+      - run: *compress_log_artifacts
+      - store_artifacts:
```

Comment:
This step is always happening after `compress_log_artifacts`. Could we also extract this like you did with `compress_log_artifacts` and re-use this code wherever needed?

Reply:
Yeah we could but I' not so sure about the `- <<:` syntax...

Reply:
Alright, it can also stay as it is now.

Reply:
The problem is that the anchor then doesn't have to reference some simple mapping but an array which needs to get merged with another array. I'm not getting any wiser from the yaml docu...